### ISSUE TITLE
Refresh Parse Keys

### ISSUE THREAD

**Issue by hjylewis** [danielmichaelni/journey#16]
@Sun77789 Can you try to refresh the Parse Application ID and Client Key?

**Comment by aubakirova** [comment]
Back to its original values? But thats gonna be visible again
btw you have access to Parse Application ID and Client Key on parse website

**Comment by hjylewis** [comment]
No, what I mean is to change the Parse Application ID and Client Key since someone could still going in to our commit history to get the keys.  We need to decommission those keys and get new ones.  Is there an easy way to do that? Usually, there is a refresh key button or something.

**Comment by aubakirova** [comment]
I can only reset master key.
But we can just create a new app on parse, put cloude code, and use those
keys as well as delete this Parse App

**Comment by hjylewis** [comment]
Okay, apparently it is fine to reveal App ID and Client Key (see https://parse.com/docs/data#security).  What we really want to do is to edit user permissions so that personal info can't be read without the master key.  I already disallowed client class creation in the app settings.  I also disabled some User permissions like Get, Find, and Create Field. I'm not sure what this might break because I can't test the app but let me know.
